src/cmd/devdraw/cocoa-screen-metal.m

```diff
@@ -558,6 +565,15 @@ - (void)resetCursorRects {
 	[self addCursorRect:self.bounds cursor:currentCursor];
 }
 
+- (void)windowDidResize:(NSNotification *)notification
+{
+	if([self inLiveResize]) {
+		LOG(@"Skip resizeimg in live resize");
```

Comment:
Is this useful? Seems like it would overwhelm the log whenever a user-driven resize is active.

Reply:
@mattwidmann Thank you for your review! I'll remove this debug Log.